time stamp_human_update
time action_taken_time
string human_action_taken
string human_belief_state
string human_real_state # the new real state that put human in after taking the action
